ui/src/components/harvest-games/HarvestGameInstructions.js

```diff
@@ -0,0 +1,357 @@
+      player: 5,
+      materials: '-',
+      points: '-',
+      imageUrl: [
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg3_1.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg3_2.gif',
+        'https://hongkong.sub.hmcc.net/wp-content/uploads/hg3_3.gif',
+      ],
+      id: 'game_3',
+      pwd: 'Airplane',
+    },
+  ];
+
+  const [selected, setSelected] = useState(0);
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = () => {
+    let pwd = document.getElementById('inputted').value;
+    setPassword(pwd);
+    localStorage.setItem(gameType[selected].id, pwd);
+    console.log(localStorage.setItem(gameType[selected].id, pwd));
+  };
+
+  const handleKeyDown = (e) => {
+    if (e.key === 'Enter') {
+      handleSubmit();
+    }
+  };
+
+  return (
+    <VStack bgColor="rgba(228, 241, 244, 1)" spacing="0">
+      <Container
+        position="start"
+        fontWeight="900"
+        bgColor="rgba(134, 185, 197, 1)"
+        fontSize={{ base: '6vw', sm: '4vw' }}
+        textColor="white"
+        marginY="0"
+        borderTopRadius={7}
+        display="flex"
+      >
+        HG-2022
+        <Image
+          marginLeft="auto"
+          w={{ base: '13vw', sm: '7.5vw' }}
+          minW={{ base: '13vw', sm: '7.5vw' }}
+          src={process.env.PUBLIC_URL + '/images/ripple.svg'}
+          alt="Logo of HMCC"
+        />
+      </Container>
+      <Container
+        bgColor="rgba(227, 224, 224, 1)"
+        textAlign="center"
+        textColor="rgba(83, 83, 83, 1)"
+        minW="100%"
+        fontWeight="500"
+        fontSize={{ base: '5vw', sm: '3vw' }}
+      >
+        Game Manual
+      </Container>
+      <Box display="flex" justifyContent="center" paddingY="2em" w="100%">
+        <Select
+          borderWidth="2px"
+          w="80%"
+          h="3em"
+          borderRadius="15"
+          borderColor="rgba(71, 108, 147, 1)"
+          textColor="rgba(71, 108, 147, 1)"
+          value={selected}
+          onChange={(e) => setSelected(e.target.value)}
+        >
+          {gameType &&
+            gameType.map((e, i) => {
+              return (
+                <option value={i} key={i}>
+                  {e.game_num}
+                </option>
+              );
+            })}
+        </Select>
+      </Box>
+      {gameType[selected].pwd === localStorage.getItem(gameType[selected].id) ||
+      gameType[selected].pwd === password ? (
+        <Box w="90%">
+          <Heading size="xl" textColor="rgba(1, 43, 85, 1)">
+            {gameType[selected].game_num}
+          </Heading>
+          <Heading
+            fontSize={{ base: '4vw', md: '2vw' }}
+            textColor="rgba(1, 43, 85, 1)"
+          >
+            {gameType[selected].name}
+          </Heading>
+          <HStack
+            justifyContent={('space-evenly', 'space-between')}
+            paddingY="5"
+            w="100%"
+            display="flex"
+          >
+            <Box
```

Comment:
make conditional rendering, only render the necessary ones if the value is defined